doc/user-manual/tools/command-line-options.rst

```diff
@@ -77,6 +77,10 @@ Generating highlighted source code
       Set URL of the CSS file used by the HTML files to
       :samp:`{URL}` (can be relative)
 
+:samp:`--html-highlight=[code,all]`
+      Whether to highlight non-Agda code as comments in
+      generated HTML files (see :ref: `generating-html`)
+
```

Comment:
Please document what the default is here.

Reply:
Ok, I'll fix that.